pyproject.toml

```diff
@@ -0,0 +1,20 @@
+[tool.poetry]
+name = "house_package_saachi"
+version = "0.1.0"
+description = "A brief description of your package"
+authors = ["Saachi Mohanty <saachi.mohanty@tigeranalytics.com>"]
+readme = "README.md"
+license = "MIT"
+homepage = "https://github.com/saachi-tiger/mle-repository"
+
+[tool.poetry.dependencies]
+python = "^3.11"
+numpy = "*"
+pandas = "*"
+scikit-learn = "*"
+matplotlib = "*"
+
+[tool.poetry.dev-dependencies]
+
+[tool.poetry.scripts]
+housing_price = 'src.house_package_saachi.housing_price:main'
```

Comment:
All these are unnecessary. Just have the very basic as per the [doc](https://setuptools.pypa.io/en/latest/userguide/quickstart.html#basic-use).
So you should be having only the following..
```
[build-system]
requires = ["setuptools"]
build-backend = "setuptools.build_meta"

[project]
name = "houseprice"
version = "0.0.1"
```

Reply:
Changed pyproject.toml accordingly